src/cloudsc_fortran_pyiface/pythonsrc/run_python_driven_cloudsc.sh

```diff
@@ -0,0 +1,8 @@
+module load python3
+python3 -m venv ~/cstest
+source ~/cstest/bin/activate
+pip install --upgrade pip
+pip install f90wrap h5py 
```

Comment:
Again, everything above the export should be dealt with during the install / make phase. For the run phase, I think we can keep a little runner script like this, although I think that the path resolution could also be done in the driver with some additional configuration.